examples/resnet_distributed_torch_scripts/setup.sh

```diff
@@ -7,10 +7,10 @@ if [ $? -eq 0 ]; then
     echo "conda env exists"
 else
     echo "conda env does not exist"
-    conda create -n resnet python=3.6 -y
+    conda create -n resnet python=3.7 -y
     conda activate resnet
 fi
-pip install -r requirements.txt
+pip install -r requirements.txt torch==1.12.1+cu113 --extra-index-url https://download.pytorch.org/whl/cu113
```

Comment:
(same Qs)

Q: why pin to 11.3, but not 11.6?

Q: does a 11.6/11.5 image support a torch cu113 wheel?

Reply:
ditto.